libraries/abstractions/ble_hal/test/src/iot_test_ble_hal_common.c

```diff
@@ -724,22 +725,25 @@ static void prvCreateCharacteristic( BTService_t * xSrvc,
     TEST_ASSERT_EQUAL( eBTStatusSuccess, xStatus );
     CharAddedComplete = true;
 
-    xStatus = IotTestBleHal_WaitEventFromQueue( eBLEHALEventCharAddedCb, NO_HANDLE, ( void * ) &xBLETESTCharCb, sizeof( BLETESTAttrCallback_t ), BLE_TESTS_WAIT );
-    TEST_ASSERT_EQUAL( eBTStatusSuccess, xStatus );
-    TEST_ASSERT_EQUAL( eBTStatusSuccess, xBLETESTCharCb.xStatus );
-
-    if( xSrvc->pxBLEAttributes[ xAttribute ].xCharacteristic.xUuid.ucType == eBTuuidType16 )
-    {
-        TEST_ASSERT_EQUAL( eBTuuidType16, xBLETESTCharCb.xUUID.ucType );
-        TEST_ASSERT_EQUAL( xSrvc->pxBLEAttributes[ xAttribute ].xCharacteristic.xUuid.uu.uu16, xBLETESTCharCb.xUUID.uu.uu16 );
-    }
-    else
+    if( EnableCb == true )
```

Comment:
Variable names are not consistent. Either use hungarian notation  `xEnableCb` like others or use camel case `enableCb` for variable names.

Reply:
changed it to bEnableCb